src/frontend/index.ts

```diff
@@ -169,6 +171,7 @@ export async function createFlowrWindow(flowrStore: Store<IFlowrStore>): Promise
     FlowrIsInitializing: () => {
       clearInterval(reloadTimeout)
       isLaunchedUrlCorrect = true
+      monitorActivity(mainWindow, flowrStore.get('flowrMonitoringTime'), reload)
```

Comment:
Every value from the store might be undefined. This is not handled in `monitorActivity`